### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by utf**
Ran black to format code again and found a few syntax bugs. I've added comments so you can see what they are.

### CODE REVIEW

File: pytaser/generator.py [hunk 1]
```diff
@@ -646,7 +644,7 @@ def generate_tas(self, energy_min, energy_max, gaussian_width=0.2, step=0.1):
                             energy_mesh_ev,
                             self.kpoint_index_down,
                             self.kpoint_weights,
-                            gaussian_width
+                            gaussian_width=gaussian_width,
```
Comment (utf) [inline pytaser/generator.py:647]: Missing comma at end of line

File: pytaser/generator.py [hunk 2]
```diff
@@ -657,7 +655,7 @@ def generate_tas(self, energy_min, energy_max, gaussian_width=0.2, step=0.1):
...
```
Comment (utf) [inline pytaser/generator.py:658]: Missing comma at end of line

File: pytaser/generator.py [hunk 3]
```diff
@@ -681,7 +679,7 @@ def generate_tas(self, energy_min, energy_max, gaussian_width=0.2, step=0.1):
         )
 
     @classmethod
-    def from_mpid(cls, api_key = None, mpid, temperature, concentration, bg=None):
+    def from_mpid(cls, mpid, temperature, concentration, bg=None, api_key=None):
```
Comment (utf) [inline pytaser/generator.py:682]: keyword arguments have to be after all positional arguments - this code wouldn't run

File: pytaser/generator.py [hunk 4]
```diff
@@ -699,10 +697,7 @@ def from_mpid(cls, api_key = None, mpid, temperature, concentration, bg=None):
             A TASGenerator class with a uniform mode bandstructure & dos object, k-weights
             and a corrected bandgap.
         """
-        if api_key is None:
-            mpr = MPRester()
-        else:
-            mpr = MPRester(api_key = api_key)
```
Comment (utf) [inline pytaser/generator.py:705]: The default value of api_key is None so no need to do this

File: setup.py [hunk 5]
```diff
@@ -40,6 +40,5 @@
         "matplotlib",
         "pymatgen>=2017.12.30",
     ],
-    },
```
Comment (utf) [inline setup.py:43]: Misplaced close bracket. This code wouldn't run!

Comment (savya10) [reply setup.py:43]: ouch lol ty for checking

**Pull request merged**